app/src/main/res/layout/fragment_geraete_new_produzent.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:fillViewport="true"
+    tools:context =".ui.geraete.geraet_new.GeraeteNewVerbraucherFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+
+        <EditText
+            android:id="@+id/geraete_new_produzent_EditVerbrauch"
+            android:layout_width="wrap_content"
+
+            android:layout_height="wrap_content"
+
+            android:layout_marginTop="8dp"
+            android:hint="z.B 30"
+            android:inputType="numberDecimal"
+            app:layout_constraintStart_toStartOf="@+id/geraete_new_produzent_TextVerbrauch"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_new_produzent_TextVerbrauch" />
+
+        <TextView
+            android:id="@+id/geraete_new_produzent_TextVerbrauch"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:text="@string/geraet_new_produzent_verbrauch"
+            app:layout_constraintStart_toStartOf="@+id/geraete_new_produzent_EditProdProJahr"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_new_produzent_EditProdProJahr" />
+
+        <TextView
+            android:id="@+id/geraete_new_produzent_TextName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="24dp"
+            android:layout_marginLeft="24dp"
+            android:layout_marginTop="16dp"
+            android:text="@string/geraet_new_TextName"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <EditText
+            android:id="@+id/geraete_new_produzent_EditName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:hint="@string/geraet_new_name"
+            android:inputType="text"
+            app:layout_constraintStart_toStartOf="@+id/geraete_new_produzent_TextName"
+            app:layout_constraintTop_toBottomOf="@+id/geraete_new_produzent_TextName" />
+
+        <TextView
+            android:id="@+id/geraete_new_prduzent_TextRaum"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:text="@string/geraet_new_TextRaum"
```

Comment:
snake_case and camelCase should not be mixed. The convention is to use snake_case within XML.

https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614

Reply:
Will also be fixed in the next commit.